doc/user/choosing-table-sizes.txt

```diff
@@ -60,7 +60,7 @@ The real full version
 
 khmer's scripts, at their heart, represents k-mers in a very memory
 efficient way by taking advantage of two data structures, `Bloom
-filters <http://en.wikipedia.org/wiki/Bloom_filter>`__ and `CountMin
+filters <http://en.wikipedia.org/wiki/Bloom_filter>`__ and `Count-Min
 Sketches <https://sites.google.com/site/countminsketch/>`__, that are
```

Comment:
http://en.wikipedia.org/wiki/Count%E2%80%93min_sketch here instead of google site - tnx.